docs/manifest.json

```diff
@@ -71,6 +71,12 @@
 		"markdown_source": "https:\/\/raw.githubusercontent.com\/WordPress\/gutenberg\/master\/docs\/extensibility/theme-support.md",
 		"parent": "extensibility"
 	},
+	{
+		"title": "Autocompletion",
```

Comment:
should we just call all these "autocomplete"?

Reply:
That sounds better to me. I can make the change.